packages/geoprocessing/scripts/aws/StackScenarioAll.test.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @group aws
+ */
+
+import * as core from "@aws-cdk/core";
+import path from "path";
+import { SynthUtils } from "@aws-cdk/assert";
+import GeoprocessingStack from "./GeoprocessingStack";
+import createTestProject from "./createTestProject";
+import fs from "fs-extra";
+
+const projectName = "all";
+const rootPath = `${__dirname}/__test__`;
+const projectPath = path.join(rootPath, projectName);
+
+afterAll(async () => {
+  await fs.remove(rootPath); // Cleanup
+});
+
+test("gp-stack-scenario-all", async () => {
+  // Stub out code asset sources expected by CDK
+  await fs.ensureDir(path.join(projectPath, ".build"));
+  await fs.ensureDir(path.join(projectPath, ".build-web"));
```

Comment:
Not sure yet if I actually need to put code files in here to be picked up.  I suspect I can assert to this level if desired.